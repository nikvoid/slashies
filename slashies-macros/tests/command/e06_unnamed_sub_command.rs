use slashies_macros::Command;

/// An command with an unnamed subcommand
#[derive(Command)]
#[name = "BadCommand"]
enum BadCommand {
    Sub(SubCommand),
}

struct SubCommand;

fn main() {}
