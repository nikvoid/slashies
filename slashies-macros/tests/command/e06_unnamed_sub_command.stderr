error: Subcommand must specify a name via the "name" attribute
 --> tests/command/e06_unnamed_sub_command.rs:7:5
  |
7 |     Sub(SubCommand),
  |     ^^^^^^^^^^^^^^^
